config/examples/sandbox_proxy.yml

```diff
@@ -25,6 +25,7 @@ development:
     - hello-world-api.3scale.net
   ignore_test_failures:
     - echo-api.3scale.net
+  apicast_registry_url: <%= ENV.fetch('APICAST_REGISTRY_URL', 'http://localhost:3000/internal/') %>
```

Comment:
So you opt for the internal server implementation.
I'd say ok :) But you should probably develop your reasoning behind your choice. Because it is necessary to add `UNICORN_WORKERS` env when launching the server for better developer experience.

Reply:
I guess this comes from the part of the description of the issue that says:

> Quick win: we can reuse the internal fake backend API https://github.com/3scale/porta/blob/master/config/routes.rb#L31-L38
> But we are coupling different independent features so I do not think it is wise

Ok, but we are already setting this data in https://github.com/3scale/porta/blob/90284d4b19a05b06cb1e945eaa2e925e2a0b30e5/config/examples/core.yml#L3 and I feel that having to set it twice is asking for problems (because it is too easy to edit it only in the `fake_core` and forget about this one). So personally I would prefer not adding a default here, and in Ruby code (perhaps in an `initializer` 😄 ) reusing the value of `System::Application.config.three_scale.core.fake_server` to set this value in the config if it is `nil`.

Reply:
And btw, it is true that we are coupling different independent features so we should create a Jira issue with label `tech-debt` explaining this if it is not going to be fixed now 😝

Reply:
After some deliberation we've come to the conclusion it's best to simply disable policies if APIcast is not available. The fake server requires too many unicorns and it does not make sense to have policies if there's no APIcast.
We can find a solution for our dev environment in a followup PR.

Reply:
> We can find a solution for our dev environment in a followup PR.

As long as it is done, I don't mind it being another PR 🤷‍♀️